src/sentry/ingest/consumer/processors.py

```diff
@@ -169,6 +171,21 @@ def process_event(
     else:
         processing_store = event_processing_store
 
+    if (
+        consumer_type == ConsumerType.Transactions or data.get("type") == "transaction"
+    ) and options.get("transactions.stale_dlq_enabled"):
+        # check timestamp
+        timestamp = message.get("timestamp") if message.get("timestamp") else start_time
+
+        if timestamp and datetime.datetime.now() - timestamp > options.get(
```

Comment:
these times probably need to be normalized

Reply:
what do you mean normalized?